publish-service/src/main/docker/README.txt

```diff
@@ -0,0 +1,16 @@
+A: Build RemRem-Publish Docker image based on RemRem-Publish from an Artifactory, e.g. Jitpack:
+cd (git root dir)/publish-service
+docker build -t remrem-publish:0.6.8 --build-arg URL=https://jitpack.io/com/github/Ericsson/eiffel-remrem-publish/publish-service/0.6.8/publish-service-0.6.8.war -f src/main/docker/Dockerfile .
```

Comment:
Same comment here, as in remrem-generate about the url.

Reply:
fixed.